core/classes/Emails/Email.php

```diff
@@ -18,39 +18,62 @@ class Email
 
     public const REGISTRATION = 1;
     public const FORGOT_PASSWORD = 3;
-    public const API_REGISTRATION = 4;
     public const FORUM_TOPIC_REPLY = 5;
     public const MASS_MESSAGE = 6;
+    public const TEST_EMAIL = 7;
 
-    /**
-     * @var array<string, string> Placeholders for email templates
-     */
-    private static array $_message_placeholders = [];
+    public static function send(User $recipient, EmailTemplate $emailTemplate)
```

Comment:
Need to keep support for setting email, Such as Store and Forms module will send emails to quests based on the email they entered